OverloaderAIDriver.lua

```diff
@@ -27,17 +27,20 @@ OverloaderAIDriver.myStates = {
 }
 
 function OverloaderAIDriver:init(vehicle)
+    --there seems to be a bug, where "vehicle" is not always set once start is pressed
+	if vehicle and vehicle:isa(Vehicle) then
```

Comment:
I don't get this, if I self.vehicle is not set, nothing is going to work. Where do we set self.vehicle? or vehicle is not a vehicle when this is first called?